include/SFML/Graphics/RenderStates.hpp

```diff
@@ -76,6 +77,14 @@ class SFML_GRAPHICS_API RenderStates
     ////////////////////////////////////////////////////////////
     RenderStates(const Transform& theTransform);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Construct a default set of render states with a custom texture coordinate type
+    ///
+    /// \param theCoordinateType Texture coordinate type to use
+    ///
+    ////////////////////////////////////////////////////////////
+    RenderStates(CoordinateType theCoordinateType);
```

Comment:
Is this constructor actually useful or is it there just to be consistent with the other ones?

Reply:
I only added it for consistency; I doubt it would be of much use on its own without providing also a texture. Would you rather remove it?

Reply:
I tend to have the mindset "if it's not useful, don't add it" as we can always add things later, but not remove them. How do others see this?